tests/actions.go

```diff
@@ -239,6 +239,8 @@ type OperatorActions interface {
 	CheckInitSQLOrDie(info *TidbClusterConfig)
 	DeployAndCheckPump(tc *TidbClusterConfig) error
 	WaitForTidbClusterReady(tc *v1alpha1.TidbCluster, timeout, pollInterval time.Duration) error
+	CheckTidbClusterHaveFailedMember(info *TidbClusterConfig, timeout, pollInterval time.Duration) error
+	CheckScaleTidbMemberToZeroReplica(info *TidbClusterConfig, timeout, pollInterval time.Duration) error
```

Comment:
If `CheckScaleTidbMemberToZeroReplica` is not common used in e2e test, I think it's unnessary to make it as an interface.